widgets/toggle.js

```diff
@@ -104,7 +104,14 @@ function prepareToggleData(transformData) {
       count: data.count
     };
 
-    return transformData && transformData(newData) || newData;
+    if (transformData) {
+      newData = transformData(newData);
+      if (typeof newData !== 'object') {
+        throw new Error('`transformData` must return an object.');
+      }
+    }
```

Comment:
Hmmm, indeed I didn't see that it was calling the `Template`. @pixelastic do we need to call `transformData` in the widget itself?